src/messages/IPostMessageSentHandler.ts

```diff
@@ -1,4 +1,4 @@
-import { IExecutionResult, IHttp, IRead } from '../accessors';
+import { IExecutionResult, IHttp, IModify, IRead } from '../accessors';
```

Comment:
@graywolf336 just a thought: would it be an option to allow both languages for the implementation. Basically, you as provider / framework could still define and implement in Typescript. It compiles to Javascript. So whoever implements a Rocketlet can either depend on the TypeScript package or the compiled Javascript - at least theoretically.